tests/rust/constant_user_defined_type.rs

```diff
@@ -0,0 +1,17 @@
+#[repr(C)]
+pub struct S {
+    field: u8,
+}
+
+/// cbindgen:enum-class=false
```

Comment:
In the future we should probably also allow paths to be constants to support regular enum classes, but this is ok for now.